package/boot/uboot-mvebu/patches/0001-cmd-mvebu-Implement-the-Marvell-hw_info-command.patch

```diff
@@ -0,0 +1,416 @@
+From e0c5472d9fbb70e6fada7f8bbae023a536dd853f Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Kerma=20G=C3=A9rald?= <gandalf@gk2.net>
```

Comment:
Are you the author of this patch? It does not seem like it to me. 

Based on this patch series:
- v2:
https://patchwork.ozlabs.org/project/uboot/patch/20210215195934.67034-2-luka.kovacic@sartura.hr/

For me, it seems like you want to have credit for something you didn't do.

- v3:
https://patchwork.ozlabs.org/project/uboot/patch/20210812233938.11633-2-luka.kovacic@sartura.hr/

This was also pointed in the mailing list for U-boot:
https://www.mail-archive.com/u-boot@lists.denx.de/msg414633.html

Reply:
I do not get any credit about another work but made a refresh of an old job, which was restarted again recently…by the original author ! Thanks to him…

Reply:
And my answer was : https://www.mail-archive.com/u-boot@lists.denx.de/msg414665.html